classifier/load_data.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import pandas as pd
+import numpy as np
+
+def load_training_test_data():
+    pass
+
+def normalize(keypoints):
+    
+    # origin keypoint
+    palmBase = np.array(keypoints[-1])
+    for k in range(len(keypoints) - 1):
+        scaled_keypoint = np.subtract(keypoints[k], palmBase)
+    
+    nom = (keypoints - keypoints.min(axis=0)) * 2
+    denom = keypoints.max(axis=0) - keypoints.min(axis=0)
+    denom[denom==0] = 1
+    
+    return -1 + nom/denom
```

Comment:
could you explain in words what this is? it's probably fine, but i just don't understand what it is by the code

Reply:
The entire thing is normalizing the columns of keypoints (so across x, y, z coordinates) between -1 and 1. Lines 15, 16 and 19 does min-max feature scaling and line 17 just makes sure we're not dividing by 0.

![image](https://user-images.githubusercontent.com/33596288/101272468-738ae980-375a-11eb-8a05-83dfbb0e2ace.png)